rasa/core/nlg/interpolator.py

```diff
@@ -52,4 +52,6 @@ def interpolate(
             else:
                 template[k] = interpolate_text(v, values)
         return template
-    return template
+    elif isinstance(template, list):
+        return [interpolate(i, values) for i in template]
+    return interpolate_text(template, values)
```

Comment:
Arriving at this point means that we don't know how to handle the type of `template`. I in this case we should just hold off and return it as it is. 
```suggestion
    return template
```